src/css/wizard-navigation-bar.scss

```diff
@@ -0,0 +1,394 @@
+  }
+
+  li {
+    &:not(:last-child):before {
+      @if ($layout == 'horizontal') {
+        @include wz-horizontal-line($width, $height, $wz-color-default);
+      }
+      @else if ($layout == 'vertical') {
+        @include wz-vertical-line($width, $height, $wz-color-default);
+      }
+    }
+
+    @if ($layout == 'vertical') {
+      a {
+        min-height: $height;
+      }
+    }
+
+    .step-indicator {
+      @if ($layout == 'horizontal') {
+        @include wz-circle-position-horizontal($width, $height, $border-width);
+      }
+      @else if ($layout == 'vertical') {
+        @include wz-circle-position-vertical($width, $height, $border-width);
+      }
+
+      $wz-indicator-state_: 'default' !global;
+      $wz-indicator-color_: $wz-color-default !global;
+      @include wz-state-circle($width, $height, $border-width);
+      @content;
+    }
+    &.optional .step-indicator {
+      $wz-indicator-state_: 'default' !global;
+      $wz-indicator-color_: $wz-color-optional !global;
+      @content;
+    }
+    &.done .step-indicator,
+    &.completed .step-indicator {
+      $wz-indicator-state_: 'default' !global;
+      $wz-indicator-color_: $wz-color-done !global;
+      @content;
+    }
+    &.current .step-indicator {
+      $wz-indicator-state_: 'default' !global;
+      $wz-indicator-color_: $wz-color-current !global;
+      @content;
+    }
+    &.current.done .step-indicator,
+    &.current.completed .step-indicator {
+      $wz-indicator-state_: 'default' !global;
+      $wz-indicator-color_: $wz-color-editing !global;
+      @content;
+    }
+
+    &.navigable a:hover .step-indicator {
+      $wz-indicator-state_: 'hover' !global;
+      $wz-indicator-color_: $wz-color-default !global;
+      @include wz-state-circle($width, $height, $border-width);
+      @content;
+    }
+    &.navigable.optional a:hover .step-indicator {
+      $wz-indicator-state_: 'hover' !global;
+      $wz-indicator-color_: $wz-color-optional !global;
+      @content;
+    }
+    &.navigable.done, &.navigable.completed {
+      a:hover .step-indicator {
+        $wz-indicator-state_: 'hover' !global;
+        $wz-indicator-color_: $wz-color-done !global;
+        @content;
+      }
+    }
+    &.navigable.current a:hover .step-indicator {
+      $wz-indicator-state_: 'hover' !global;
+      $wz-indicator-color_: $wz-color-current !global;
+      @content;
+    }
+    &.navigable.current.done, &.navigable.current.completed {
+      a:hover .step-indicator {
+        $wz-indicator-state_: 'hover' !global;
+        $wz-indicator-color_: $wz-color-editing !global;
+        @content;
+      }
+    }
+  }
+}
+
+
+// Define navigation bar layout
+//
+// Arguments:
+//   $layout - 'horizontal' | 'vertical'
+//
+@mixin wz-define-layout($layout) {
+  aw-wizard-navigation-bar.#{$layout} {
+
+    &.small ul.steps-indicator {
+      @include wz-define-style($layout, $width: $wz-small-dot-width, $height: $wz-small-dot-height, $border-width: 0) {
```

Comment:
Wrapped `@include`s? (What does the inner `@content` do in combination with the outer one?)

Reply:
There is nothing special here. A content block passed to the outer mixin (`wz-define-style`) happens to include another mixin (`wz-state-circle-with-background`). I guess, you can think of it as passing one mixin into another.

Reply:
Yes together with your above comments it becomes clear.